tests/testCommandBasedBackendUnified.cpp

```diff
@@ -5,21 +5,22 @@
 // Define a name for the test module.
 #define BOOST_TEST_MODULE CommandBasedBackend
 // Only after defining the name include the unit test header.
-#include <boost/test/unit_test.hpp>
-using namespace boost::unit_test_framework;
 
 #include "DummyServer.h"
 
 #include <ChimeraTK/UnifiedBackendTest.h>
 
 using namespace ChimeraTK;
 
+#include <boost/test/unit_test.hpp>
```

Comment:
Why change the place of this include? Was this done my the auto code formater? At least the comment that goes with it should be moved along.